index.html

```diff
@@ -0,0 +1,464 @@
+                  </li>
+                  <li>If |route to| is [=route to/existing-client-navigate=], run the steps to [=navigate=] |client| passing |target URL|.
+                  </li>
+                  <li>Return |client|.
+                  </li>
+                </ol>
+            </dl>
+          </li>
+        </ol>
+      </section>
+    </section>
+    <section>
+      <h2>
+        Script Interfaces to App Launches
+      </h2>
+      <section data-dfn-for="LaunchParams">
+        <h3>
+          <dfn>LaunchParams</dfn> interface
+        </h3>
+        <ul class="issue">
+          <li>This has been copied directly from <a href="https://wicg.github.io/manifest-incubations/index.html#launch-queue-and-launch-params">Manifest Incubations</a> without modification, this [=manifest/launch_handler=] spec should be its replacement home.</li>
+          <li>{{LaunchParams}} should be a dictionary.</li>
+          <li>{{LaunchParams/targetURL}} should not be nullable.</li>
+          <li>{{LaunchParams/files}} should be optional</li>
+          <li>For members other than {{LaunchParams/targetURL}} only one should be not undefined, indicating the type of launch.</li>
+        </ul>
+        <pre class="idl">
+          [Exposed=Window] interface LaunchParams {
+            readonly attribute DOMString? targetURL;
+            readonly attribute FrozenArray&lt;FileSystemHandle&gt; files;
+          };
+        </pre>
+        <p>
+          {{LaunchParams/targetURL}} represents the [=URL=] of the launch which
+          MUST be [=manifest/within scope=] of the application.
+        </p>
+        <p>
+          For every |file handle:FileSystemHandle| in {{LaunchParams/files}},
+          querying the file system permission with
+          {{FileSystemPermissionDescriptor/mode}} set to
+          {{FileSystemPermissionMode/"readwrite"}} MUST return
+          {{PermissionState/"granted"}}.
+        </p>
+      </section>
+      <section data-dfn-for="LaunchConsumer">
+        <h3>
+          <dfn>LaunchConsumer</dfn> function
+        </h3>
+        <pre class="idl">
+          callback LaunchConsumer = any (LaunchParams params);
+        </pre>
+      </section>
+      <section data-dfn-for="LaunchQueue">
+        <h3>
+          <dfn>LaunchQueue</dfn> interface
+        </h3>
+        <pre class="idl">
+          partial interface Window {
+            readonly attribute LaunchQueue launchQueue;
+          };
+
+          [Exposed=Window] interface LaunchQueue {
+            undefined setConsumer(LaunchConsumer consumer);
+          };
+        </pre>
+        <p>
+          {{LaunchQueue}} has an <dfn>unconsumed launch params</dfn> which is a
+          [=list=] of {{LaunchParams}} that is initially empty.
+        </p>
+        <p>
+          {{LaunchQueue}} has an <dfn>assigned launch consumer</dfn> which is
```

Comment:
"... which is a {{LaunchConsumer}} that is ..."

Reply:
Done.